public/app/features/plugins/extensions/constants.ts

```diff
@@ -1 +1 @@
-export const MAX_EXTENSIONS_PER_PLACEMENT_PER_PLUGIN = 2;
+export const MAX_EXTENSIONS_PER_PLUGIN = 2;
```

Comment:
```suggestion
export const MAX_EXTENSIONS_PER_POINT = 2;
```

Reply:
Fixed in https://github.com/grafana/grafana/pull/65841/commits/5e82147cfc9befb9df02527ff292c6d1df6dbcfb.